apps/api/src/merchants/dto/create-merchant.dto.ts

```diff
@@ -85,4 +86,7 @@ export class CreateMerchantDto {
   @IsNotEmpty()
   @IsPhoneNumber()
   readonly phoneNumber: string;
+
+  @IsBoolean()
+  readonly isActive: boolean;
```

Comment:
Remove this no need to pass isActive when creating

Reply:
removed